message-me/app/javascript/packs/application.js

```diff
@@ -12,9 +12,17 @@ import '@doabit/semantic-ui-sass'
 Rails.start()
 Turbolinks.start()
 ActiveStorage.start()
+
+window.scroll_bottom = function() {
+  if ($('#messages').length > 0) {
+    $('#messages').scrollTop($('#messages')[0].scrollHeight);
+  }
+}
```

Comment:
Why not:
```suggestion
window.scroll_bottom = function(id) {
  const ele = $(`#${id}`)
  const scrollHeight = ele[0].scrollHeight;
  ele.scrollTop();
}
```

Reply:
it can display many lines of messages, so everytime new message entered, the view should be at the bottom

Reply:
bottom of page and last message is 2 very different requirements. which?

Reply:
it is targeting the messages div, which is the chatbox. So go to the bottom of the chatbox/last message.